aten/src/ATen/core/jit_type.h

```diff
@@ -736,7 +919,10 @@ struct CAFFE2_API TupleType : public Type {
 
   bool compare(const Type& rhs, std::function<bool(const TypePtr, const TypePtr)> fn) const {
     if(rhs.kind() != kind())
+    {
```

Comment:
While I appreciate that you fix this, I'd prefer this to be committed separately (along with a similar fix `s/bool numel/size_t numel/` above and other cleanups if any).